docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -22,3 +22,6 @@ General configuration properties
     - Cache the fact that metadata, including table and column statistics, is
       not available
     - False
+  * - ``allow-drop-table``
```

Comment:
We are going to remove this: https://github.com/trinodb/trino/pull/588

Reply:
Removed from this PR.